src/frontend/src/service/currency-format.test.ts

```diff
@@ -0,0 +1,13 @@
+import currencyFormat from './currency-format.ts';
+describe('CURRENCY FORMAT SERVICE TEST', () => {
+  it('formats currency', () => {
+    var str = currencyFormat(145);
```

Comment:
Recommend using `const` instead of `var`
Also I'd suggest renaming the variable to `currencyString`

Reply:
I reuse the str here.

Reply:
I'll rename it though.